### DISCUSSION TITLE
how do I upgrade?

### DISCUSSION THREAD

**Discussion by AntonioSun** [smicyk/groovy-jmeter#85] [Q&A] [answered]
Following up with https://github.com/smicyk/groovy-jmeter/issues/72#issuecomment-1150637197

> First steps   
To run your test script, it is enough to type ...   
Please remember that first execution of the script can take some time as Ivy must download all dependency to local cache

Would everything get upgraded by itself next time I run it when there is a new release?  
How can I verify that my local cache has the latest groovy-jmeter?

**Comment by AntonioSun** [comment, marked as answer]
> Would everything get upgraded by itself next time I run it when there is a new release?

Indeed:

> In general it should update automatically. There might be some period between release on github and public maven repositories (everything needs to propagate). You can try this command to see if you get the list of jars for the release:
> 
> ```shell
> grape resolve net.simonix.scripts groovy-jmeter 0.18.0
> ```

**Reply by AntonioSun** [reply]
Seems that my `@Grab('net.simonix.scripts:groovy-jmeter')` is still giving me v0.17.0, yet a new release is available at https://github.com/smicyk/groovy-jmeter/releases/tag/v0.18.0

**Reply by smicyk** [reply]
In general it should update automatically. There might be some period between release on github and public maven repositories (everything needs to propagate). You can try this command to see if you get the list of jars for the release:

```shell
grape resolve net.simonix.scripts groovy-jmeter 0.18.0
```

**Reply by AntonioSun** [reply]
> In general it should update automatically. There might be some period between release on github and public maven repositories (everything needs to propagate). You can try this command to see if you get the list of jars for the release:
> 
> ```shell
> grape resolve net.simonix.scripts groovy-jmeter 0.18.0
> ```

Thanks! That works!!
